grails-app/views/order/_summary.gsp

```diff
@@ -71,7 +71,42 @@
             <g:elseif test="${orderInstance?.shipments}">
                 <g:set var="disabledMessage" value="${g.message(code:'order.errors.rollback.message')}"/>
             </g:elseif>
-            <g:if test="${currentState}">
+            <g:if test="${orderInstance?.orderTypeCode == OrderTypeCode.TRANSFER_ORDER}">
```

Comment:
@jmiranda Are the Transfer Order the only order type that had this issue. I see that we have in the system 3 other order types. Are those used anywhere? Other than that LGTM

Reply:
I don't think any of the other order types have been implemented yet.